lmdeploy/pytorch/kernels/dipu/multinomial_sampling.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Since this implementation is pure pytorch, we can place it in default instead of dipu.

Reply:
According to your design, it would be more appropriate to place it in the default folder, I would put it in the default folder in the next commit.